src/actions/projects.js

```diff
@@ -116,6 +121,18 @@ const getAvatarsForProjects = projects => {
     })
 }
 
+const getMuseumRoleForProjects = projects => {
+    return projects.map(project => {
+        return apiClient.type('project_roles')
+            .get({id: project.links.project_roles})
+            .then((roles) => {
```

Comment:
Hmmm. Good point. This is kind of confusing.

The `roles.roles` thing comes from our API, _but_ you bring up a good point. It might be possible for us to name these passed-in arguments in some way that stays true to the objects we are receiving while preventing some confusion.